src/backend/metal/src/lib.rs

```diff
@@ -713,7 +713,11 @@ struct PrivateCapabilities {
     max_texture_layers: u64,
     max_fragment_input_components: u64,
     sample_count_mask: u8,
+<<<<<<< HEAD
```

Comment:
that's an unresolved merge conflict :)

Reply:
I have no idea how to do that 'squashing' thing :(